client/jetpack-connect/authorize.js

```diff
@@ -692,7 +689,7 @@ export class JetpackAuthorize extends Component {
 	}
 }
 
-export default connect(
+const connectComponent = connect(
```

Comment:
Bit of a nitpick: `connectComponent` sounds a bit like a verb/action, so I think using the participle  (`connectedComponent`) would be more descriptive in cases like this.

Reply:
It is a verb 🙂 

It's "partially applied," connect is `connect( mapStateToProps, mapDispatchToProps )( WrappedComponent )`. Here it is a function ready to receive a component to connect.

Reply:
:man_facepalming: You're obviously right @sirreal.

Ignore me please :grimacing:

Reply:
Yup, what Jon said 😉 Our `connect` calls here return a function, and calling _that_ function connects the component. Which essentially justifies the verb.

Reply:
> Ignore me please 😬

I never do 😛